src/OrigamiGovernanceToken.sol

```diff
@@ -158,6 +192,10 @@ contract OrigamiGovernanceToken is
 
     /// @dev this is overridden so we can apply the `whenNotPaused` modifier
     function _beforeTokenTransfer(address from, address to, uint256 amount) internal override whenNotPaused {
+        (uint256 lockedAmount, uint256 deadline) = getTransferLock(from);
+        if (deadline > 0 && balanceOf(from) >= amount && balanceOf(from) - amount < lockedAmount) {
+            require(block.timestamp > deadline, "TransferLock: address lockup has not expired");
+        }
```

Comment:
Minor comment: If they have some amount that isn't locked but is less than the amount attempting to be transferred, the error message could maybe be improved by including that as a possible reason. Maybe something like "TransferLock: address lockup has not expired OR unlocked amount is insufficient". I do realize they could call `getTransferLock` for that information as well so I leave it for you to decide.

Reply:
Great point and thanks for the callout! What do you think of the rephrased version? `solhint` wants the reason string to be less than 32 characters... this fits that and I think still conveys the intent.

Reply:
I like it! :shipit: